certified-connectors/JG Integrations/apiDefinition.swagger.json

```diff
@@ -0,0 +1,519 @@
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "pattern": {
+                  "type": "string",
+                  "description": "pattern",
+                  "title": "Regex Pattern"
+                },
+                "replacement": {
+                  "type": "string",
+                  "description": "replacement",
+                  "title": "Replacement Value",
+                  "default": null
+                },
+                "subject": {
+                  "type": "string",
+                  "description": "subject",
+                  "title": "Value to Search"
+                }
+              },
+              "required": [
+                "pattern",
+                "subject"
+              ]
+            }
+          }
+        ],
+        "description": "Replace all characters matched by REGEX Expression"
+      }
+    },
+    "/manual_func": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "JSON Response",
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            },
+            "schema": {
+              "type": "object",
+              "properties": {
+                "output": {
+                  "type": "string",
+                  "description": "Processed value",
+                  "title": "Processed Value"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Advanced PHP Functions",
+        "x-ms-visibility": "advanced",
+        "description": "Manually specify a php function and body",
+        "operationId": "MANUAL",
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "Accept",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "application/json",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "function": {
+                  "type": "string",
+                  "description": "function",
+                  "title": ""
+                },
+                "data": {
+                  "type": "string",
+                  "description": "data",
+                  "title": ""

```

Comment:
@jglassonnz Empty property values can be removed until unless you have a specific reason to keep them.